setup.py

```diff
@@ -39,7 +39,7 @@
     },
     test_suite='tests',
     install_requires=[
-        'lxml<4.7.1',
+        'lxml<=4.9.0',
```

Comment:
I think you meant `>= 4.9.0` or `~= 4.9`? It makes no sense to allow 4.9.0 but disallow 4.9.1.

Reply:
I am adding pin to avoid any breaking change in any upcoming release. Just update the pin as per new release.

Reply:
Pinning the version like this does not work well for projects that use `python3-saml` because in order to use a new version of `lxml` you have to wait for a new release of `pyton3-saml`. After this change users still can't update `lxml` to get security fixes easily.